homeassistant/config.py

```diff
@@ -453,3 +455,31 @@ def merge_packages_customize(core_customize, packages):
         conf = schema(pkg)
         cust.extend(conf.get(CONF_CORE, {}).get(CONF_CUSTOMIZE, []))
     return cust
+
+
+@asyncio.coroutine
+def async_check_ha_config_file(hass):
+    """Check if HA config file valid.
+
+    This method is a coroutine.
+    """
+    import homeassistant.components.persistent_notification as pn
```

Comment:
It's a hack but one that we use in other places in this file too, see [line 31](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/__init__.py#L31). However, line 31 does execute it a bit more classy, as it allows for people to overwrite that component.

I prefer to use the function over spelling out the service, because that's the suggested way for Python code.